### DISCUSSION TITLE
safari中油猴加载此插件，无法识别已安装到本地的霞鹜文楷

### DISCUSSION THREAD

**Discussion by katower** [F9y4ng/GreasyFork-Scripts#170] [问答专区(Question / Answer)] [answered]
如题，已将霞鹜文楷安装到字体册中，但是插件无法识别到，导致备选字体中不出现霞鹜文楷

**Comment by F9y4ng** [comment]
1. 系统安装新字体后需要重启浏览器，才能使字体在浏览器中生效。为确保字体缓存正确重建，重启浏览器后，需在高级核心设置中**重建**字体缓存。
2. 如果你已确认安装了正确的字体文件，以及字体（中、英）名称都无误的情况下，很可能是Safari不能通过字体名称识别该字体。

比如：**微软雅黑** 字体的英文名称在safari下就是无法识别的，这与浏览器内核对字体的解析有关。

**Reply by F9y4ng** [reply, marked as answer]
在safari引入第三方字体可参考 #260 的方法。
